tests/infer/mcmc/test_nuts.py

```diff
@@ -175,7 +173,7 @@ def model(data):
     true_probs = torch.tensor([0.9, 0.1])
     data = dist.Bernoulli(true_probs).sample(sample_shape=(torch.Size((1000,))))
     nuts_kernel = NUTS(model, step_size, adapt_step_size, adapt_mass_matrix, full_mass)
-    mcmc_run = MCMC(nuts_kernel, num_samples=500, warmup_steps=100).run(data)
+    mcmc_run = MCMC(nuts_kernel, num_samples=400, warmup_steps=200).run(data)
```

Comment:
the test for `adapt_mass_matrix=True` and `adapt_step_size=False` is slow; so I increase warmup_steps to force NUTS learn a better mass_matrix.